backend/src/main/java/com/social/dailylink/service/PostServiceImpl.java

```diff
@@ -26,10 +28,6 @@ public PostServiceImpl(AbstractEntityRepository<Post> repository, UserRepository
 
     @Override
     public Post create(Post entity) {
```

Comment:
There is a prePersist Method in the AbstractEntityService for this perpose

Reply:
Ich han kei ahnig was presave macht 😀

Reply:
Das wird immer vor em Save usgfüert. Chasch ja mal d'Implementation vo save im AbstractEntityServiceImpl aluege